test/tests.js

```diff
@@ -140,7 +140,7 @@ describe('IPFS Node.js API wrapper tests', function () {
           throw err
         }
         var added = res[res.length - 1]
-        assert.equal(added.Hash, 'QmaMTzaGBmdLrispnPRTESta4yDQdK4uKSVcQez2No4h6q')
+        assert.equal(added.Hash, 'QmZdsefMGMeG6bL719gX44XSVQrL6psEgRZdw1SGadFaK2')
```

Comment:
because the output changed since 0.3.7-- the output of add now has one last hash which is the root of them all.